src/cdk-experimental/scrolling/virtual-scroll-viewport.spec.ts

```diff
@@ -15,7 +15,9 @@ describe('CdkVirtualScrollViewport', () => {
         imports: [ScrollingModule, ExperimentalScrollingModule],
         declarations: [AutoSizeVirtualScroll],
       }).compileComponents();
+    });
 
+    beforeEach(() => {
```

Comment:
async?

Reply:
Everything in this block is synchronous

Reply:
Whoops I meant to say the one above should be async